packages/sdk-react/src/components/counterparts/CounterpartsTable/Filters/Filters.tsx

```diff
@@ -32,95 +25,89 @@ export const Filters = ({ onChangeFilter, showCategories }: Props) => {
   const className = 'Monite-CounterpartFilters';
 
   return (
-    <Grid
-      container
-      spacing={2}
-      className={classNames(className, 'Monite-Filters')}
-    >
-      <Grid item xs={4} sm={5} md={5} lg={4}>
+    <FilterContainer
+      className={className}
+      searchField={
         <SearchField
           label={t(i18n)`Search by name`}
           onChange={(search) => {
             onChangeFilter(FILTER_TYPE_SEARCH, search);
           }}
         />
-      </Grid>
-      <Grid item xs={4} sm={3} md={3}>
+      }
+    >
+      <FormControl
+        variant="outlined"
+        fullWidth
+        className="Monite-CounterpartTypeFilter Monite-FilterControl"
+      >
+        <InputLabel id="type">{t(i18n)`Type`}</InputLabel>
+        <Select
+          labelId="type"
+          label={t(i18n)`Type`}
+          defaultValue="all"
+          MenuProps={{ container: root }}
+          onChange={(search) => {
+            onChangeFilter(FILTER_TYPE_TYPE, search.target.value);
+          }}
+        >
+          {[
+            { label: t(i18n)`All`, value: 'all' },
+            {
+              label: t(i18n)`Individuals`,
+              value: 'individual',
+              icons: <PersonIcon color="primary" fontSize="small" />,
+            },
+            {
+              label: t(i18n)`Companies`,
+              value: 'organization',
+              icons: <BusinessIcon color="success" fontSize="small" />,
+            },
+          ].map(({ label, value }) => (
+            <MenuItem value={value} key={value}>
+              {/* We should use `ListItemIcon` component to be able to show `icons` */}
+              <Box sx={{ marginLeft: 1 }}>{label}</Box>
+            </MenuItem>
+          ))}
+        </Select>
+      </FormControl>
+      {showCategories && (
         <FormControl
           variant="outlined"
           fullWidth
-          className="Monite-CounterpartTypeFilter Monite-FilterControl"
+          className="Monite-CounterpartCategoryFilter Monite-FilterControl"
         >
-          <InputLabel id="type">{t(i18n)`Type`}</InputLabel>
+          <InputLabel id="category">{t(i18n)`Category`}</InputLabel>
           <Select
-            labelId="type"
-            label={t(i18n)`Type`}
+            labelId="category"
+            label={t(i18n)`Category`}
             defaultValue="all"
             MenuProps={{ container: root }}
             onChange={(search) => {
-              onChangeFilter(FILTER_TYPE_TYPE, search.target.value);
```

Comment:
As I see, this refactoring with the selectors `onChangeFilter` and `value`s should be a part for separate PR. Moreover, this refactoring the code without need adds difficulties for other developers on merges.
Btw, I would not change options to `'false' | 'true'`